src/app/component/ProductActions/ProductActions.container.js

```diff
@@ -278,8 +278,21 @@ export class ProductActionsContainer extends PureComponent {
         } = variants[configurableVariantIndex] || product;
 
         if (type_id === BUNDLE) {
-            const { price_range: { minimum_price: { regular_price: { currency } } } } = product;
-            const priceValue = { value: selectedBundlePrice, currency };
+            const {
+                price_range: {
+                    minimum_price: {
+                        regular_price: { currency },
+                        discount: { percent_off }
+                    }
+                }
+            } = product;
+
+            const finalBundlePrice = selectedBundlePrice === 0
+                ? selectedBundlePrice
+                // eslint-disable-next-line no-magic-numbers
+                : selectedBundlePrice - (selectedBundlePrice * (percent_off / 100));
```

Comment:
If selectedBundlePrice is 0, then `selectedBundlePrice * (percent_off / 100)` will result 0, and `0 - 0` gives 0.
Please, remove the unnecessary ternary operator.